server.lua

```diff
@@ -158,15 +166,21 @@ RegisterNetEvent('qb-storerobbery:server:successsafe', function()
     local Player = QBCore.Functions.GetPlayer(source)
     local PlayerCoords = GetEntityCoords(GetPlayerPed(source))
     local ClosestSafeIndex = GetClosestSafe(PlayerCoords)
+    local markedBills = math.random(Config.SafeReward.MarkedBillsAmount.Min, Config.SafeReward.MarkedBillsAmount.Max)
 
     if not ClosestSafeIndex then return end
     if not StartedSafe[source] then return end
 
-    local Info = {
-        worth = math.random(Config.SafeReward.MarkedBillsWorth.Min, Config.SafeReward.MarkedBillsWorth.Max)
-    }
-    Player.Functions.AddItem('markedbills', math.random(Config.SafeReward.MarkedBillsAmount.Min, Config.SafeReward.MarkedBillsAmount.Max), false, Info)
-    TriggerClientEvent('inventory:client:ItemBox', source, QBCore.Shared.Items['markedbills'], 'add')
+    for i = 1, markedBills do
+        local worth = math.random(Config.SafeReward.MarkedBillsWorth.Min, Config.SafeReward.MarkedBillsWorth.Max)
+
+        local Info = {
+            worth = worth,
+            description = Lang:t('text.value', { value = worth })
+        }
+
+        Player.Functions.AddItem('markedbills', 1, false, Info)
```

Comment:
Why does each marked bill have a worth? If you are already giving a random number of marked bills, it seems simpler to have them all be worth the same and the randomness is how many they get. This is the first I've seen something that suggests different bill denominations are supported, which is why I'm surprised to see the existing metadata code.

Reply:
My idea here, which I think it's what the old QBCore code was doing, I changed this a while ago but hadn't PR'd it, is that you get different bags with different amounts, so that it plays more with chance on if you get out with lot of money or just an ok amount. Trying to make it more interesting for players. What do you think?

Reply:
That should be a separate PR I think. It's difficult for me to understand how that relates to the code, as there are no bags here. Just bills that have a random metadata amount value, which seems odd to me.

Reply:
I get what you mean now, I think I misunderstood the way it worked. I just updated the PR with the conversion so that it is just as it was but with ox_inventory, I believe, let me know if you think that's fine or we should still move it to another PR.

Reply:
What you have is good. It preserves the existing behavior. The existing behavior is still strange, is the worth metadata isn't being used anywhere? In any case, we can explore this more in subsequent PRs.